db.go

```diff
@@ -42,10 +42,11 @@ func init() {
 	db.Exec(`
 		CREATE TABLE IF NOT EXISTS client (
 			id INT UNSIGNED NOT NULL AUTO_INCREMENT,
-			country_code VARCHAR (50) NOT NULL,
-			phone_number VARCHAR (50) NOT NULL,
-			verification_code VARCHAR (50),
-			PRIMARY KEY (id)
+			country_code VARCHAR(50) NOT NULL,
```

Comment:
country_code VARCHAR (50) NOT NULL, // consistent spacing

Reply:
I was looking at [MySQL](https://dev.mysql.com/doc/refman/5.7/en/char.html) doc & saw that they write with no spacing for data types. But, now I think their spacing is kinda arbitrary too.

I agreed on consistent spacing 0b3dc1a43a5cfc2af5851c908f42405a20dbb103.